src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
```

Comment:
Be consistent with your margins. Here you're adding margin-right to each `.stars__star` element, and then resetting it for the last child. It's better to add only bottom or top margin to maintain consistency and avoid potential margin collapse issues.